SICP_book/chapter_2_1.metta

```diff
@@ -178,44 +216,30 @@
 ; work using either representation?
 
 
-; First representation of rect will be four points (top-left, top-right, bottom-left, bottom-right.
-
-; This variant doesn't work for me:
-;(= (make-rect $top-left $top-right $bottom-left $bottom-right)
-;  (cons $bottom-left (cons $top-left (cons $top-right $bottom-right))))
-
-; Using this variant, simple_rect looks like this:
-; [((1 . 2) cons-atom (1 . 3) ((2 . 3) 2 . 2))]
+; First representation of rect will be four points (top-left, top-right, bottom-left, bottom-right).
 
-; So instead, I wrote make-rect this way:
+(= (cons-atom' $head $tail) (cons-atom $head $tail))
 
 (= (make-rect $top-left $top-right $bottom-left $bottom-right)
-    (let $right (cons-atom $top-right $bottom-right)
-        (let $top_and_right (cons-atom $top-left $right)
-            (cons-atom $bottom-left $top_and_right))))
+  (cons-atom' $bottom-left (cons-atom' $top-left (cons-atom' $top-right ($bottom-right)))))
```

Comment:
Minor, but you can also just form a 4-positions tuple: `($bottom-left $top-left $top-right $bottom-right)`
```suggestion
  ($bottom-left $top-left $top-right $bottom-right)
```
I also not sure if `cons-atom'` is needed for `make-rect2`